docs/IntegrationWithExistingApps.md

```diff
@@ -573,8 +573,12 @@ AppRegistry.registerComponent('HelloWorld', () => HelloWorld);
 ## Prepare your current app
 
 In your app's `build.gradle` file add the React Native dependency:
-
-    compile "com.facebook.react:react-native:+"  // From node_modules
+```
+dependencies {
+    ...
+    compile "com.facebook.react:react-native:+" // From node_modules. Replace + with actual React Native version.
```

Comment:
@satya164 I guess it's a matter of personal preference. I always like to be explicit.

Changed into a note.